ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # Revision history for capability
 
+## 0.2.0.0 -- 2019-03-22
+
+* Make HasStream a superclass of HasWriter.
+  See [#64](https://github.com/tweag/capability/pull/64)
```

Comment:
And compatibility with recent `generic-lens` releases?

Reply:
Done. I've added a line for it.